PurchasesCoreSwift/Logging/Strings/PurchaseStrings.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PurchaseStrings.swift
+//  PurchasesCoreSwift
+//
+//  Created by Tina Nguyen on 12/11/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCPurchaseStrings) public class PurchaseStrings: NSObject {
+    @objc public var cannot_purchase_product: String { "Could not purchase SKProduct. There is a problem with your configuration in App Store Connect. More info here: http://errors.rev.cat/products-empty" }  //appleWarning
+    @objc public var entitlements_revoked: String { "Entitlements revoked for product identifiers: %@. \nsyncing purchases" } //debug
```

Comment:
`entitlements_revoked_syncing_purchases`, there should never be a message the developer doesn't expect when reading the class